integreat_cms/cms/templates/feedback/region_feedback_list.html

```diff
@@ -184,9 +184,12 @@ <h1 class="heading">
                                 {% translate "Delete" %}
                             </option>
                         {% endif %}
-                        <option data-bulk-action="{% url 'export_region_feedback' region_slug=request.region.slug %}">
+                        <option data-bulk-action="{% url 'export_region_feedback' region_slug=request.region.slug file_format='csv' %}">
                             {% translate "Export to CSV" %}
                         </option>
+                        <!--<option data-bulk-action="{% url 'export_region_feedback' region_slug=request.region.slug file_format='xlsx' %}">
```

Comment:
```suggestion

```
Since we agreed that xlsx shouldn't be supported, we can remove this and the comment below